documentation/book/ref-metrics.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-metrics.adoc
+
+[id='ref-metrics-{context}']
+= Metrics configuration
+
+Prometheus metrics can be enabled by configuring the `metrics` property in following resources:
+
+* `Kafka.spec.kafka`
+* `Kafka.spec.zookeeper`
+* `KafkaConnect.spec`
+* `KafkaConnectS2I.spec`
+
+When the `metrics` property is not defined in the resource, the Prometheus metrics will be disabled.
+To enable Prometheus metrics export without any further configuration, you can set it to an empty object (`{}`).
```

Comment:
This seems more like a procedure than reference. I'm okay with us entering an issue to turn this into a procedure post-Beta.

Reply:
Well ... that is one of the things which is IMHO bit unfortunate ... sometimes the line between procedure and reference is very thin. If it makes it better, we have procedure for this as well, so it is IMHO ok: `proc-configuring-metrics.adoc`.